ddm_test_validate.py

```diff
@@ -0,0 +1,214 @@
+import logging
+import datetime
+import pathlib
+import os
+import time
```

Comment:
unused imports. we can probably remove?

Reply:
Actually these are used for logging in line 49-62:
`if log_file=="enable":
            current_time = datetime.datetime.now().strftime("%Y-%m-%d-%H-%M-%S")
            log_file = current_time + "_" + env_name + "_log.csv"
            log_file2 = current_time + "_" + "SIM" + "_log.csv"
            logs_directory = pathlib.Path(log_file).parent.absolute()
            if not pathlib.Path(logs_directory).exists():
                print(
                    "Directory does not exist at {0}, creating now...".format(
                        str(logs_directory)
                    )
                )
                logs_directory.mkdir(parents=True, exist_ok=True)
        self.log_file = os.path.join(log_path, log_file)
        self.log_file2 = os.path.join(log_path, log_file2 `

Reply:
Removed `time`, that was used during tests only, but not really necessary.